bindings/cpp/src/lib.rs

```diff
@@ -65,10 +73,17 @@ mod ffi {
         fn remove(self: &Operator, path: &str) -> Result<()>;
         fn stat(self: &Operator, path: &str) -> Result<Metadata>;
         fn list(self: &Operator, path: &str) -> Result<Vec<Entry>>;
+        fn reader(self: &Operator, path: &str) -> Result<Box<Reader>>;
+
+        fn fill_buf(self: &mut Reader) -> Result<&[u8]>;
```

Comment:
The naming could be confused with functions of `Operator`, how about using `reader_fill_buf`?

Reply:
The exposed functions have already bound with corresponding c++ type. For example, the reader will call function by `reader.fill_buf()`, rather than `fill_buf(&reader)` like c. So I think using `reader_fill_buf` is unnecessary.